docs/pages/blog/2021-developer-survey-results.md

```diff
@@ -0,0 +1,406 @@
+  <tr><td>73</td><td>design - consistency</td><tr>
+  <tr><td>73</td><td>design - Material Design</td><tr>
+  <tr><td>6</td><td>design - ?</td><tr>
+  <tr><td>4</td><td>design - easy</td><tr>
+  <tr><td>2</td><td>design - responsiveness</td><tr>
+  <tr><td>1</td><td>design - time</td><tr>
+  <tr><th>309</th><th>components</th><tr>
+  <tr><td>173</td><td>components - quantity</td><tr>
+  <tr><td>124</td><td>components - quality</td><tr>
+  <tr><td>5</td><td>components - advanced</td><tr>
+  <tr><td>3</td><td>components - data grid</td><tr>
+  <tr><td>2</td><td>components - ?</td><tr>
+  <tr><td>1</td><td>components - autocomplete</td><tr>
+  <tr><td>1</td><td>components - base/unstyled</td><tr>
+  <tr><th>306</th><th>DX</th><tr>
+  <tr><td>221</td><td>DX - easy to use</td><tr>
+  <tr><td>49</td><td>DX - consistency</td><tr>
+  <tr><td>32</td><td>DX - API</td><tr>
+  <tr><td>4</td><td>DX - ?</td><tr>
+  <tr><th>181</th><th>customizability</th><tr>
+  <tr><th>57</th><th>docs</th><tr>
+  <tr><td>39</td><td>docs - ?</td><tr>
+  <tr><td>17</td><td>docs - quality</td><tr>
+  <tr><td>1</td><td>docs - quantity</td><tr>
+  <tr><th>26</th><th>community</th><tr>
+  <tr><th>24</th><th>accessibility</th><tr>
+  <tr><th>17</th><th>performance</th><tr>
+  <tr><td>15</td><td>performance - runtime</td><tr>
+  <tr><td>1</td><td>performance - bundle size</td><tr>
+  <tr><td>1</td><td>performance - ?</td><tr>
+  <tr><th>9</th><th>community support</th><tr>
+  <tr><th>5</th><th>icons</th><tr>
+  <tr><th>3</th><th>typescript</th><tr>
+  <tr><th>2</th><th>animations</th><tr>
+</table>
+</details>
+
+### Please rate how much you agree or disagree with the following statements
+
+<img src="/static/blog/2021-survey/4.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="I can find most of the components I need (Strongly agree: 45.5%, Agree: 45.9%, Neutral: 6.2%, Disagree: 1.5%, Strongly Disagree 0.8%); I can easily customize the components to match the desired design (Strongly agree: 23.4%, Agree: 46.1%, Neutral: 18.6%, Disagree: 9%, Strongly Disagree 2.9%); I can find the answers to most of my questions in the documentation (Strongly agree: 24.1%, Agree: 50.1%, Neutral: 16.3%, Disagree: 7.6%, Strongly Disagree 1.9%); I find the library's performance to be great (Strongly agree: 26.4%, Agree: 44.3%, Neutral: 22.6%, Disagree: 4.7%, Strongly Disagree 2%); Whenever I needed to get help, I received helpful responses, StackOverflow or GitHub (Strongly agree: 20.9%, Agree: 36.7%, Neutral: 36%, Disagree: 4.6%, Strongly Disagree 1.8%)" />
+
+<p class="blog-description">1534 out of 1589 answered</p>
+
+It's definitely a good thing to have a general sense of agreement to most of the statements. However, there's a considerably big gap between _Strongly agree_ and _Agree_ for all but the first statement, which showcases that we need to continue pushing for even more quality in each of these dimensions (customizability, customization, performance, and support).
```

Comment:
I believe this is the first time in the piece where the potential responses are italicized. Do we want to remove the italics here, or should we instead italicize them throughout?

Reply:
I have italicized a few other instances just for consistency's sake. Is there a more correct approach?